kibana/Dockerfile

```diff
@@ -9,13 +9,13 @@
 # Build stage 0 `builder`:
 # Extract Kibana artifact
 ################################################################################
-FROM centos:8 AS builder
+FROM  AS builder
```

Comment:
@tylersmalley @jbudz Hi, it seems the docker-build-context tarball file is not correctly generated.
Could you please help on the values to put here.
thanks

Reply:
PR at https://github.com/elastic/kibana/pull/122852.  I'll update here in a few minutes with the build contexts generated there.

I'll follow up with an issue for us to smoke test in our repo.

Reply:
Opened https://github.com/elastic/kibana/issues/122865 to make sure we start testing these.

Reply:
@jbudz we also need to update line 12, is that `FROM ubuntu:20.04`?

Reply:
@jbudz got the following error
```
/bin/sh: 1: curl: not found
The command '/bin/sh -c cd /tmp &&   curl --retry 8 -s -L     --output kibana.tar.gz      https://artifacts.elastic.co/downloads/kibana/kibana-8.0.0-rc1-linux-$(arch).tar.gz &&   cd -' returned a non-zero code: 127
```